features/collections/collection_navigation.feature

```diff
@@ -71,4 +71,20 @@ Feature: Basic collection navigation
     And "A League of Their Own" should appear before "Merlin"
     And "Merlin" should appear before "Teen Wolf"
 
-
+  Scenario: Collections can be filtered by media type
+    Given I have the collection "We all sing together"
+      And I have a canonical "TV Shows" fandom tag named "Steven's Universe"
+      And I have a canonical "Movies" fandom tag named "High School Musical"
+    When I am logged in as "Brian" with password "They called him Brian"
+      And I post the work "Stronger than you" with fandom "Steven's Universe" in the collection "We all sing together" 
+      And I post the work "Breaking Free" with fandom "High School Musical" in the collection "We all sing together"
+      And I go to "We all sing together" collection's page
+      And I follow "Fandoms ("
+      And I choose "Movies" as the type
```

Comment:
It might be better to just stick with the basic `I select "OPTION" from "#medium_id"` here and `I select "OPTION" from "Media"` for the unassigned fandoms test. It's a little ugly here because you have to refer to the select by its ID, but the names of the steps are so vague and their purpose is so context-specific, creating a step definition doesn't add much value.

If you _really_ wanted to create a step, I'd suggest making it cover a bunch of the navigation-related steps, like:

```
When/^I view the "([^"]*)" medium in the collection "([^"]*)"$/ do |medium, collection|
  visit collection_fandoms_path(Collection.find_by_title(collection))
  select(medium, from: "medium_id")
  click_button "Show"
end
```